docs/paper/dev/guides/pdc.md

```diff
@@ -0,0 +1,148 @@
+---
+slug: /dev/pdc
+---
+
+# Persistent Data Container (PDC)
+
+The Persistent Data Container (PDC) is a way to store custom data on a whole range of objects; such as items, entities, and blocks.
+The Full list of classes that support the PDC are:
+
+- [Chunk](#chunk)
+- [World](#world)
+- [Entity](#entity)
+- [TileState](#tilestate)
+- [ItemMeta](#itemmeta)
+- [Structure](#structure)
+
+## What is it used for?
+In the past, developers resorted to storing data within the NBT tags of an object. 
+This was a very hacky way of storing data, and was prone to having conflicts between different plugins
+when they used the same keys. This is resolved by the use of `NamespacedKey`.
+Therefore, the PDC is a much more stable way of storing data, and is much easier to use.
+
+As the name implies, the PDC is persistent. This means that the data stored in the PDC will be saved to disk, 
+and will be loaded back into the object when the server is restarted. This is great for storing attributes of an object,
+such as the owner of an item, or the custom name of an entity.
+
+## Adding Data
+To store data in the PDC, there are a few things you need first. The first is a `NamespacedKey` which is used to identify the data.
+The second is a `PersistentDataContainer` which is the object you want to store the data on. The third is the data itself.
+
+```java
+// Create a NamespacedKey
+NamespacedKey key = new NamespacedKey(pluginInstance, "example-key");
+
+ItemStack item = new ItemStack(Material.DIAMOND);
+// ItemMeta implements PersistentDataHolder so we can get the PDC from it
+ItemMeta meta = item.getItemMeta();
+item.getPersistentDataContainer().set(key, PersistentDataType.STRING, "I love Tacos!");
+item.setItemMeta(meta);
+```
+
+:::note
+
+It is considered good practice to reuse the `NamespacedKey` objects. Constructing them from a static context is ideal for this.
+
+:::
+
+## Getting Data
+To get data from the PDC, you need to know the `NamespacedKey` and the `PersistentDataType` of the data.
+
+```java
+// Create a NamespacedKey
+NamespacedKey key = new NamespacedKey(pluginInstance, "example-key");
+
+ItemStack item = ...; // Retrieve the item from before
+// Get the data from the PDC
+PeristentDataContainer container = item.getItemMeta().getPersistentDataContainer();
```

Comment:
Small typo PeristentDataContainer instead of PersistentDataContainer

Reply:
Thanks <3